src/styles/sidebar/components/datalist.scss

```diff
@@ -0,0 +1,75 @@
+@use "../../variables" as var;
+
+.datalist {
+  position: relative;
+}
+
+.datalist__items {
+  &:before {
+    /* creates a small outlined triangle above the drop down menu */
+    background-color: inherit;
+    border: inherit;
+    border-top-left-radius: 0.125rem;
+    clip-path: polygon(0 0, 100% 0, 0% 100%, 0% 100%);
+    content: '';
+    display: inline-block;
+    height: 0.5rem;
+    left: 0.5rem;
+    position: absolute;
+    top: -0.3125rem;
+    transform: rotate(45deg);
+    width: 0.5rem;
+  }
+  position: absolute;
+  font-size: var.$body2-font-size;
+  top: 5px;
+  max-width: 100%;
+  min-width: 10em;
+  background-color: var.$white;
+  border: 1px solid var.$grey-3;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.1);
+  z-index: 10;
+
+  @media (pointer: coarse) {
+    font-size: var.$touch-input-font-size;
+  }
+}
+
+.datalist__ul {
+  list-style: none;
+  padding: 0;
+  > li {
+    padding: 0.25em 1.5em 0.25em 1em;
+    border-left: 4px transparent solid;
```

Comment:
On mobile the list items are quite short vertically if you are trying to tap them. Can you add a `@media (pointer: coarse) { ... }` query to make sure that the items are at least `var.$touch-target-size` in height on mobile?